README.md

```diff
@@ -2,3 +2,5 @@
 Linux Management Platform
 
 short description: http://fo0.me/index.php/posts/linux-management-platform
+
+test
```

Comment:
bitte in sämtlichen sprachen hinterlegen

Reply:
@en3rgy97 mach mal